classes/controllers/grid/GridHandler.inc.php

```diff
@@ -876,16 +876,6 @@ protected function renderFilter($request, $filterData = array()) {
 		switch(true) {
 			case $form === null: // No filter form.
 				return '';
-			case is_a($form, 'Form'): // Form object subclass
```

Comment:
I think this case is still used (grep around for subclasses of `ClientFormHandler`).

Reply:
@asmecher, pointers? I don’t see `ClientFormHandler` anywhere in the code base, except in JavaScript. I looked pretty thoroughly at usage before deleting that clause.

Reply:
Just grepped a little further and it looks like you're right. I think the submission lists used to use this case but they've been adapted to vue.js. However, I think you still need to change the self-doc for GridHandler::getFilterForm.